pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -187,6 +188,41 @@ func (p *podSchedulingState) publish(ctx context.Context, pod *v1.Pod, clientset
 			logger.V(5).Info("Updating PodSchedulingContext", "podSchedulingCtx", klog.KObj(schedulingCtx))
 		}
 		_, err = clientset.ResourceV1alpha2().PodSchedulingContexts(schedulingCtx.Namespace).Update(ctx, schedulingCtx, metav1.UpdateOptions{})
```

Comment:
Have you considered a regular patch? Is that any faster than SSA?

Reply:
I haven't because the guidance is to use SSA. I've shared CPU and memory profiles with @apelisse, he is looking into the performance aspect.